c3d/src/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: test
+
+ifeq (${CLANG_BUILD},)
+$(error "Please define $$CLANG_BUILD, it should point to your clang build directory")
+endif
+
+test:
+	make -C out/
+	${CLANG_BUILD}/bin/clang -mllvm -debug-only=c3d -std=c2x -fplugin=./out/C3d.so ../tests/basic0.h
```

Comment:
can you ifdef on the platform, e.g.

```
ifeq($(OS),Windows_NT)
SO = dll
else
  ifeq($(uname -s),Darwin)
    SO = dylib
  else
    SO = so
  endif
endif
```

or something along those lines?

Reply:
Sure, will do!